app/Providers/TwilioProvider.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+namespace App\Providers;
+
+use App\TaskRouter\WorkspaceFacade;
+use Illuminate\Support\Facades\Log;
+use Illuminate\Support\ServiceProvider;
+use Twilio\Rest\Client;
+use Twilio\Rest\Taskrouter;
+
+class TwilioProvider extends ServiceProvider
+{
+    /**
+     * Bootstrap the application services.
+     *
+     * @return void
+     */
+    public function boot()
+    {
+        //
+    }
+
+    /**
+     * Register the application services.
+     *
+     * @return void
+     */
+    public function register()
+    {
+        $this->app->singleton(Client::class, function ($app) {
```

Comment:
I think singleton in this case is not really necessary, specially as PHP will create a new instance everytime it gets a request